client/src/pages/recording-view/components/plugins-pane.tsx

```diff
@@ -44,10 +45,15 @@ export const PluginsPane = () => {
   const [modalSamples, setModalSamples] = useState<Float32Array>(new Float32Array([]));
   const [modalSpectrogram, setmodalSpectrogram] = useState(null);
   const [useCloudStorage, setUseCloudStorage] = useState(true);
-  const { dataSourcesQuery } = useUserSettings();
-  const connectionInfo = dataSourcesQuery?.data[`${account}/${container}`];
+  const dataSource = getDataSource(CLIENT_TYPE_API, account, container, true).data;
```

Comment:
We need to be careful this page might still be used with a blob datasource or with a local file so pining up this here might not be a good idea